src/applications/mhv-landing-page/components/alerts/AlertMhvBasicAccount.jsx

```diff
@@ -2,29 +2,29 @@ import React, { useEffect } from 'react';
 import PropTypes from 'prop-types';
 // eslint-disable-next-line import/no-named-default
 import { default as recordEventFn } from '~/platform/monitoring/record-event';
+import { logoutUrl } from 'platform/user/authentication/utilities';
 import { logoutUrlSiS } from '~/platform/utilities/oauth/utilities';
-import { VaAlert } from '@department-of-veterans-affairs/component-library/dist/react-bindings';
+import {
+  VaAlert,
+  VaButton,
+} from '@department-of-veterans-affairs/component-library/dist/react-bindings';
 
-const MhvBasicAccountAlert = ({ headline, recordEvent, status }) => {
+const AlertMhvBasicAccount = ({ headline, recordEvent, testId, ssoe }) => {
   useEffect(() => {
     recordEvent({
       event: 'nav-alert-box-load',
       action: 'load',
       'alert-box-headline': headline,
-      'alert-box-status': status,
+      'alert-box-status': 'warning',
     });
   }, []); // eslint-disable-line react-hooks/exhaustive-deps
 
   const signOut = () => {
-    window.location = logoutUrlSiS;
+    window.location = ssoe ? logoutUrl() : logoutUrlSiS();
```

Comment:
This is/was a bug. Initially, the function for determining the signout url wasn't being executed (lack of parens: `logoutUrlSiS()`).

Updated the code to follow what the [PersonalizationDropdown does](https://github.com/department-of-veterans-affairs/vets-website/commit/35230ddf3e3fd9369dc5ba3430d071fe36e82cb3#diff-11d2008deaf42fd28d385ca6f5a46e28d5965ea2d16a3ccce68797ac1f218e28R28) for signing out.